unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/EntityIdHelper.cs

```diff
@@ -50,7 +55,7 @@ public long EntityFromLegacyEntityString(string entityId)
         public long GetConvertedEntityId(string entityId)
         {
             long entityIdLong = 0;
-            var regex = new Regex("^*[A-Z]");
+  
             string entityIdExtracted = entityId.Substring(1);
```

Comment:
The regex is only for one character. Could we change it for something like:

```
const string base36CharList = "0123456789abcdefghijklmnopqrstuvwxyz"; // (maybe in uppercase)
char entityIdExtracted = entityId.Substring(1);

if (base36CharList.Contains(entityIdExtracted))
```

Reply:
This regex is working for all characters, just tested it!

Reply:
However, Tomorrow the approach will change we will optimize this  and for that we must remove regex